src/themes/five/src/components/Checkout/index.js

```diff
@@ -465,7 +461,7 @@ const CheckoutUI = (props) => {
           <CartContainer>
             <CartHeader>
               <h1>{t('MOBILE_FRONT_YOUR_ORDER', 'Your order')}</h1>
-              <span onClick={() => cart?.business?.slug && handleGoToStore(cart?.business?.slug)}>{('ADD_PRODUCTS', 'Add products')}</span>
+              <span onClick={() => cart?.business?.slug && handleStoreRedirect(cart?.business?.slug)}>{('ADD_PRODUCTS', 'Add products')}</span>
```

Comment:
Please validate if `handleStoreRedirect` is null

Ex: `handleStoreRedirect && handleStoreRedirect(...)`

Reply:
added validation